style.css

```diff
@@ -0,0 +1,770 @@
+}
+
+.image {
+  position: relative;
+}
+
+.img {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background-color: rgba(235, 47, 47, 0.4);
+}
+
+.write p {
+  margin-top: -40px;
+}
+
+@media (min-width: 992px) {
+  body {
+    width: 100%;
+  }
+
+  .toggle {
+    display: none;
+  }
+
+  .head-builder {
+    margin: 0;
+    font-size: 3em;
+  }
+
+  .main-container {
+    width: 100%;
+  }
+
+  .mainImgTech {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .firstImg,
+  .secondImg {
+    width: 43%;
+  }
+
+  .first-header {
+    display: flex;
+    background-color: #272a31;
+    color: white;
+    justify-content: end;
+    margin-left: auto;
+    margin-top: auto;
+    font-size: 18px;
+    padding-top: 13px;
+    padding-bottom: 13px;
+    padding-right: 50px;
+    font-family: "Poppins", sans-serif;
+  }
+
+  .top-menu {
+    padding-right: 30px;
+    height: 20px;
+  }
+
+  .second-nav {
+    display: flex;
+    align-items: center;
+    position: relative;
+    width: 100%;
+    height: auto;
+    background-color: aliceblue;
+    padding: 0;
+    padding-top: 8px;
+    margin: 0;
+    margin-top: -13px;
+  }
+
+  .close {
+    display: none;
+  }
+
+  .displayf {
+    flex-direction: row;
+    justify-content: end;
+    padding: 0;
+    align-items: center;
+    gap: 20px;
+    width: 80%;
+  }
+
+  .displayf li {
+    list-style: none;
+    font-size: 18px;
+    font-family: "Poppins", sans-serif;
+    padding: 0;
+    margin: 0;
+    color: #272a31;
+  }
+
+  .logo__ul {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 85%;
+  }
+
+  .logo {
+    margin-right: 150px;
+    margin-left: 0;
+    font-family: "Poppins", sans-serif;
+    margin-top: 0;
+    font-weight: bold;
+    font-size: 24px;
+    padding-left: 50px;
+    color: black;
+  }
+
+  .camp {
+    color: red;
+    font-size: 20px;
+    font-family: "Poppins", sans-serif;
+    border: 5px solid red;
+    width: 100%;
+    margin: 0;
+    padding: 0 5px;
+    text-align: center;
+  }
+
+  .campcc {
+    display: flex;
+    justify-content: end;
+    border: none;
+    margin-left: 40px;
+    width: 20%;
+  }
+
+  .intro-section p {
+    margin-left: 110px;
+    font-size: 50px;
+    margin-top: 60px;
+  }
+
+  .intro-section .dozer {
+    font-size: 60px;
+    font-family: "Poppins", sans-serif;
+    color: #ec5242;
+    margin-top: auto;
+    margin-left: 110px;
+    width: 700px;
+  }
+
+  .box p {
+    font-size: 28px;
+    width: 900px;
+    border: 3px solid whitesmoke;
+  }
+
+  .date h3 {
+    padding-left: 110px;
+    font-size: 40px;
+    font-family: "lato", sans-serif;
+    padding-top: 20px;
+  }
+
+  .date p {
+    margin-top: 10px;
+    font-size: 20px;
+  }
+
+  /* .cards {
+    display: flex;
+    flex-direction: row;
+    padding-top: 100px;
+    padding-bottom: 100px;
+    width: 90%;
+    margin: auto;
+    justify-content: center;
+  } */
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
Kindly remove all commented code in your style.css file 😊